client/packages/system/src/Patient/hooks/usePatientStore.ts

```diff
@@ -0,0 +1,12 @@
+import { zustand } from '@openmsupply-client/common';
+
+interface PatientStoreState {
+  documentName: string | undefined;
+  setDocumentName: (documentName: string | undefined) => void;
+}
```

Comment:
This is for the current patient document or any document? Maybe a comment would be good here...